HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/JobDetail.cs

```diff
@@ -6,6 +6,7 @@ namespace HelpMyStreetFE.Models.Account.Jobs
 {
     public class JobDetail
     {
+        public RequestSummary RequestSummary { get; set; }
```

Comment:
Does this model class need to be defined here when it's already defined in the nuget package

Reply:
I think RequestSummary here is the one from NuGet.